tests/cypress/e2e/unit_tests/rbac_fleet.spec.ts

```diff
@@ -16,87 +16,66 @@ import 'cypress/support/commands';
 import * as cypressLib from '@rancher-ecp-qa/cypress-library';
 import { qase } from 'cypress-qase-reporter/dist/mocha';
 
+export const baseUser      = "base-user"
+export const uiPassword    = "rancherpassword"
+export const roleTypeTemplate = "Global"
```

Comment:
Same as earlier, but up to you

Reply:
`roleTypeTemplate` we can keep it at global level as it is used in most of the test cases. Please let me know if you want me to remove from here and have added in individual test cases.